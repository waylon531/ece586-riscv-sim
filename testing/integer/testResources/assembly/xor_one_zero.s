addi t0, zero, 1
addi t1, zero, 0
xor t2, t1, t0 ; t2 should contain 1