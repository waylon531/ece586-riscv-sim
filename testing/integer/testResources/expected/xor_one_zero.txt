PC:0x0000000C
RA:0x00000000
SP:0x00000000
GP:0x00000000
TP:0x00000000
T0:0x00000001
T1:0x00000000
T2:0x00000001
S0:0x00000000
S1:0x00000000
A0:0x00000000
A1:0x00000000
A2:0x00000000
A3:0x00000000
A4:0x00000000
A5:0x00000000
A6:0x00000000
A7:0x00000000
S2:0x00000000
S3:0x00000000
S4:0x00000000
S5:0x00000000
S6:0x00000000
S7:0x00000000
S8:0x00000000
S9:0x00000000
S10:0x00000000
S11:0x00000000
T3:0x00000000
T4:0x00000000
T5:0x00000000
T6:0x00000000